### ISSUE TITLE
FR: New arrow type

### ISSUE THREAD

**Issue by CTJoriginal** [zsviczian/obsidian-excalidraw-plugin#1691]
**Problem:** A lot of times when annotating plots I want to anootate really narrow area that is too small for regular arrow ( <---> ) to fit in. 

**Solution:** Since recently many new arrow heads were added, i thought it might be possible to  Inversed arrow head: ->---<- 
[Example](https://github.com/zsviczian/obsidian-excalidraw-plugin/assets/48687492/f29557f7-da92-4bfb-9bcb-3857316215ea)

Alternatively I can draw arrow each time but that takes up a lot of time....

**Comment by zsviczian** [comment]
makes sense. I might add a feature allowing you to define custom arrow types... however, those arrows will not work if you send your file to someone or paste your drawing to Excalidraw.com
